src/game/Chat/Level3.cpp

```diff
@@ -1399,615 +1399,611 @@ bool ChatHandler::HandleCooldownClearArenaCommand(char*)
-        "11417",
-        "11422",
-        "11419",
-        "11424",
-        "11420",
-        "27",
-        "31",
-        "33",
-        "34",
-        "35",
-        "15125",
-        "21127",
-        "22950",
-        "1180",
-        "201",
-        "12593",
-        "12842",
-        "16770",
-        "6057",
-        "12051",
-        "18468",
-        "12606",
-        "12605",
-        "18466",
-        "12502",
-        "12043",
-        "15060",
-        "12042",
-        "12341",
-        "12848",
-        "12344",
-        "12353",
-        "18460",
-        "11366",
-        "12350",
-        "12352",
-        "13043",
-        "11368",
-        "11113",
-        "12400",
-        "11129",
-        "16766",
-        "12573",
-        "15053",
-        "12580",
-        "12475",
-        "12472",
-        "12953",
-        "12488",
-        "11189",
-        "12985",
-        "12519",
-        "16758",
-        "11958",
-        "12490",
-        "11426",
-        "3565",
-        "3562",
-        "18960",
-        "3567",
-        "3561",
-        "3566",
-        "3563",
-        "1953",
-        "2139",
-        "12505",
-        "13018",
-        "12522",
-        "12523",
-        "5146",
-        "5144",
-        "5148",
-        "8419",
-        "8418",
-        "10213",
-        "10212",
-        "10157",
-        "12524",
-        "13019",
-        "12525",
-        "13020",
-        "12526",
-        "13021",
-        "18809",
-        "13031",
-        "13032",
-        "13033",
-        "4036",
-        "3920",
-        "3919",
-        "3918",
-        "7430",
-        "3922",
-        "3923",
-        "7411",
-        "7418",
-        "7421",
-        "13262",
-        "7412",
-        "7415",
-        "7413",
-        "7416",
-        "13920",
-        "13921",
-        "7745",
-        "7779",
-        "7428",
-        "7457",
-        "7857",
-        "7748",
-        "7426",
-        "13421",
-        "7454",
-        "13378",
-        "7788",
-        "14807",
-        "14293",
-        "7795",
-        "6296",
-        "20608",
-        "755",
-        "444",
-        "427",
-        "428",
-        "442",
-        "447",
-        "3578",
-        "3581",
-        "19027",
-        "3580",
-        "665",
-        "3579",
-        "3577",
-        "6755",
-        "3576",
-        "2575",
-        "2577",
-        "2578",
-        "2579",
-        "2580",
-        "2656",
-        "2657",
-        "2576",
-        "3564",
-        "10248",
-        "8388",
-        "2659",
-        "14891",
-        "3308",
-        "3307",
-        "10097",
-        "2658",
-        "3569",
-        "16153",
-        "3304",
-        "10098",
-        "4037",
-        "3929",
-        "3931",
-        "3926",
-        "3924",
-        "3930",
-        "3977",
-        "3925",
-        "136",
-        "228",
-        "5487",
-        "43",
-        "202",
-        "0"
+    static std::vector<uint32> allSpellList
```

Comment:
Should std::array be used instead maybe? And ofc the same on gmSpellList

Reply:
Agreed

Reply:
change it and i will merge this this week. I promise.

Reply:
I fixed it for you